libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -52,26 +56,199 @@ typedef struct TFModel{
     TF_Graph *graph;
     TF_Session *session;
     TF_Status *status;
+    SafeQueue *request_queue;
+    Queue *inference_queue;
+    Queue *task_queue;
 } TFModel;
 
+/**
+ * Stores execution parameters for single
+ * call to the TensorFlow C API
+ */
+typedef struct tf_infer_request {
+    TF_Output *tf_outputs;
+    TF_Tensor **output_tensors;
+    TF_Output *tf_input;
+    TF_Tensor *input_tensor;
+} tf_infer_request;
```

Comment:
ffmpeg's name style:  TFInferRequest

Reply:
Yes, actually, I was a bit confused about what to use for its name. Thanks.